README.md

```diff
@@ -0,0 +1,18 @@
+# Home Assistant Yellow website
+
+This is the source of the Yellow documentation website at [yellow.home-assistant.io](https://yellow.home-assistant.io). 
+
+## Setup
+
+This website is built using [11ty](https://www.11ty.dev/). The website is automatically deployed using Netlify.
+
+## Development
```

Comment:
Might it be an idea to by default point to codespaces?

Especially if one would just make a single edit quickly, you don't need anything and just start typing working directly from your browser...